app/com/linkedin/drelephant/exceptions/ExceptionFinder.java

```diff
@@ -64,37 +71,50 @@ public ExceptionFinder(String url, String scheduler)
 
     // create a new MRClient
     _mrClient = new MRClient();
+    logger.info("mr client");
```

Comment:
Please change to Debug ,as you are just printing static string

Reply:
this wasn't meant to be added, included by mistake.